### DISCUSSION TITLE
Integrate several libraries into one collection

### DISCUSSION THREAD

**Discussion by Dvx84** [meisnate12/Plex-Meta-Manager#460] [Q&A]
Friends,

First, let me thank you so much for the great work done here. This is a truly great add-on to Plex and I love the new version.

I managed to get everything working and am very happy with the results, however, I am struggling with one question as I’m kind of a new with this.

I have three libraires in with my movies are located (e.g. movie1, movie2, movie3). I know what to create a collection for IMDB Top250 and want the movies of all three libraries to be integrated in that. 

Is that possible? If yes, how do I need to adjust the config yml? Can I add several library names into my first library Movie, i.e.

Movie:
Library_name:movie1
Library_name:movie2
Library_name:movie3

Or do I need to set this up differently? 

Thanks a ton,

DvX

**Comment by meisnate12** [comment]
so plex doesnt allow collections like that what you can do is create a collection with the same name in each library and the other libraries will show up below the current library movies but it wouldn't be very ideal in your situation. Also since plex doesn't do true multi library collections, PMM only works with one library at a time for a collection

**Reply by Dvx84** [reply]
Hey man,

Thanks again for your answer. 

The great news is, that it seems that Plex fixed it themselves. Since I use smart.labels it seems that Plex is able to look for the identical label across my different libraries. When I look into the collection, it then shows me the remaining top 250 movies for example that have been associated to the respective label in the other libraries.

The only think one needs to do is run the same collection creation script for each library and Plex does the rest. Really cool.

Thanks again for that really cool script

**Reply by levander051** [reply]
> so plex doesnt allow collections like that what you can do is create a collection with the same name in each library and the other libraries will show up below the current library movies but it wouldn't be very ideal in your situation. Also since plex doesn't do true multi library collections, PMM only works with one library at a time for a collection

is it possible to set up multiple libraries in config file and if yes how?

**Reply by Droppisalt** [reply]
If you mean multiple libraries like Movies, TV Shows, Anime, etc., then yes. See here:

https://metamanager.wiki/en/latest/config/libraries.html
